pkgs/tools/admin/chkservice/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, cmake, ninja, pkg-config, systemd, ncurses, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "chkservice";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "linuxenko";
+    repo = "chkservice";
+    rev = version;
+    hash = "sha256:0dfvm62h6dwg18f17fn58nr09mfh6kylm8wy88j00fiy13l4wnb6";
+  };
+
+  # Tools needed during build time
+  nativeBuildInputs = [
+    cmake
+    # Makes the build faster, adds less than half a megabyte to the build
+    # dependencies
```

Comment:
do we want to keep these comments?

Reply:
I'd argue that specifying any build runner (e.g. GNU Make, Ninja) next to cmake is to be expected and does not require additional justification.
I'd only be interested to know why if something exotic happened, e.g. more than one was specified.

Reply:
I think if we remove the comment we should also remove ninja. It's not a dependency of this package, if at all it should be a dependency of cmake itself. This comment gives an explanation as to why ninja is there even though it's not a dependency.